docs-src/modules/getting-started/pages/index.adoc

```diff
@@ -1,14 +1,141 @@
+Kaskada is built on simple, powerful abstrctions that may be different than what you're accustomed to.
+
+If you'd prefer to jump right in, scroll down to the xref:#quick-starts[] section
+====
+
+== Thinking with Kaskada's timelines
+
+Kaskada is built on the idea of a _timeline_ - the history of how a value changes over time for a specific entity or group.
+
+[stream_viz,name=basic-sum]
+.Aggregating events as a timeline
+....
+[
+    {
+        "label": "Purchase.amount",
+        "kind": "discrete",
+        "data": [
+            [
+                {"t": 2, "v": "5"},
+                {"t": 5, "v": "2"},
+                {"t": 13, "v": "13"},
+                {"t": 18, "v": "4"}
+            ]
+        ]
+    },
+    {
+        "label": "Purchase.amount | sum()",
+        "kind": "continuous",
+        "data": [
+            [
+                {"t": 2, "v": "5"},
+                {"t": 5, "v": "7"},
+                {"t": 13, "v": "20"},
+                {"t": 18, "v": "24"}
+            ]
+        ]
+    }
+]
+....
+
+Timelines give you declarative queries and the ability to reason about temporal context, time travel, sequencing, time-series, and more.
+Transforming and combining timelines allow you to intuitively express computations over events.
+
+****
+⭢ Read more about timelines in xref:overview:what-is-kaskada.adoc[]
+****
+
+With Kaskada, it's timelines all the way down - every operation starts from timelines and produces timelines. 
+When you make a query, the resulting timeline is interpreted in one of two ways: as a _history_ or as a _snapshot_. 
+
+* A timeline *History* generates a value each time the timeline changes, and each row is associated with a different entity and point in time. 
+* A timeline *Snapshot* generates a value for each entity at the _same_ point in time; each row is associated with a different entity, but all rows are associated with the same time.
+
+These values may be output in different ways, for example they can be written as rows in a Parquet file, or sent as events to a stream.
+
+****
+⭢ Read more about converting to tables in xref:developing:queries.adoc#configuring-how-queries-are-computed[Queries]
+****
+
+== Kaskada's basic architecture
+
+Kaskada is implemented as a standalone service with two main components.
+
+* The *engine* is a stateless compute engine for transforming event data.
+* The *manager* is an (optionally) stateful API for communicating with clients.
+
+These components are distributed as standalone, precompiled binaries.
+The Kaskada service is accessed using a gRPC API, and multiple clients exist.
+You can access the API using the provided Python client or using the CLI binary.
+Kaskada can be configured to run as a remote service or as a local process.
+
+****
+⭢ Read more about installing Kaskada in xref:ROOT:installing.adoc[]
+****
+
+== Kaskada's data model
+
+Kaskada stores events in xref:developing:tables.adoc[tables]. 
```

Comment:
Connect this to timelines somehow. Ideally, starting with timelines:

```
Input timelines come from tables.
Tables consist of multiple events, and each event is a value of the same type.
```

Alternatively, introduce tables and relate back:

```
...
Each input table describes a source timeline which may be used in queries.
```